_includes/js/schema.md

```diff
@@ -0,0 +1,145 @@
+# Schema
+
+Schema is the structure representing classes in your app. You can use the schema
+of an app to verify operations in a unit test, generate test data, generate test
+classes and then clean up after tests. The schema API can also be used to create
+custom views of your data. We use the schema API to display columns names and
+types in the databrowser.
+
+This API allows you to access the schemas of your app.
+
+* `MasterKey` is required.
+* Starting with Parse-Server 2.7.1
+
+Schema will return an object similar to the following:
+
+<pre><code class="javascript">
+{
+  className: 'MyClass',
+  fields: {
+    //Default fields
+    ACL: {type: 'ACL'},
+    createdAt: {type: 'Date'},
+    updatedAt: {type: 'Date'},
+    objectId: {type: 'String'},
+    //Custom fields
+    aNumber: {type: 'Number'},
+    aString: {type: 'String'},
+    aBool: {type: 'Boolean'},
+    aDate: {type: 'Date'},
+    aObject: {type: 'Object'},
+    aArray: {type: 'Array'},
+    aGeoPoint: {type: 'GeoPoint'},
+    aPolygon: {type: 'Polygon'},
+    aFile: {type: 'File'}
+  },
+  classLevelPermissions: {
+    find: {
+      '*': true
+    },
+    create: {
+      '*': true
+    },
+    get: {
+      '*': true
+    },
+    update: {
+      '*': true
+    },
+    addField: {
+      '*': true
+    },
+    delete: {
+      '*': true
+    }
+  },
+  indexes: {
+    indexName: { aString: 1 },
+  }
+}
+</code></pre>
+
+Direct manipulation of the classes that are on your server is possible through ParseSchema. Although fields and classes can be automatically generated (the latter assuming client class creation is enabled) ParseSchema gives you explicit control over these classes and their fields.
```

Comment:
Probably would be good to add a little warning with this as well.
> With great power comes great responsibility. Altering the schema directly should be done with care, you can't go back to retrieve data if you remove a field and it's associated values.

Reply:
I like